test/suites/optional/generic-device-plugin.robot

```diff
@@ -49,6 +49,29 @@ Verify ttyUSB glob pattern device discovery and allocation
 
     [Teardown]    TtyUSB Glob Test Teardown
 
+Verify FUSE device allocation and accessibility
+    [Documentation]    Verifies FUSE device configuration, allocation, and accessibility in pods
+    [Tags]    fuse-device
+    Enable And Configure GDP    ${GDP_CONFIG_FUSE_COUNT}
+    Wait Until Device Is Allocatable    10    fuse
+    # Create and deploy pod requesting 4 FUSE devices
+    ${path}=    Create Random Temp File    ${POD_FUSE_DEVICE}
+    Oc Create    -f ${path} -n ${NAMESPACE}
+
+    # Wait for pod to be ready and verify device accessibility
+    Oc Wait    -n ${NAMESPACE} pod/fuse-test-pod    --for=condition=Ready --timeout=120s
+
+    # Verify /dev/fuse is accessible in the pod
+    ${fuse_device}=    Run With Kubeconfig    oc exec -n ${NAMESPACE} fuse-test-pod -- ls -l /dev/fuse
```

Comment:
We have `Oc Exec` keyword, can we use it instead?

Reply:
Done.